setup.py

```diff
@@ -81,6 +81,8 @@ def run(self):
             where=".",
             include=["cunumeric*"],
         ),
+        package_data={"cunumeric": ["sphinxext/_templates/*.rst"]},
```

Comment:
I assume this is to allow the new sphinx extension to fetch the template from the right place, right? I find this a bit odd as these files are used only when building the doc, which by the way also requires other rst files. Maybe we should include the entire doc source files in the package?

Reply:
Yes that is the reason. Personally I would keep things as they are, as the most minimal compromise that's good for everyone. I doubt any end users are going to want to build the docs, so including those files is both a packaging size and complexity increase that seems unjustified. The "clean" solution I guess would be to split `cunumeric.sphinxext` off to a separate package but I am strongly :-1: as it makes like more difficult for devs that want to build the docs (FWIW in general I am on "team monorepo"). I guess an alternative would be to make this private `cunumeric._sphinxext`

Reply:
Keeping only those files in the package sounds reasonable. Let's give it an internal module name to avoid confusion.

Reply:
https://github.com/nv-legate/cunumeric/pull/323/commits/6a37212c4f1f8e4398e8282a42cbe6d701a9d3f1